pkg/porter/cnab.go

```diff
@@ -331,10 +329,9 @@ func (o *sharedOptions) combineParameters(ctx context.Context) map[string]string
 	}
 
 	//
-	// Default the porter-debug param based on the verbosity level
+	// Default the porter-debug param to --debug
```

Comment:
Did we change the code back to the original code here?

Reply:
Yes, per our discussion last week we are going to keep a --debug flag on the commands that run bundles. The PORTER_DEBUG parameter should be set off of that and not the verbosity flag. I have a follow-up PR after this to change how we check debug based on the new flag but it required more refactoring.

Reply:
OK, if you look at an individual commit you may see changes (because I'm avoiding a force push while you are reviewing) but if you diff the PR as a whole, this file is no longer modified. That was really confusing! 😅

Reply:
Ohh, yes, I was looking through each individual commits. Thanks for the clarification!